packages/fields/src/Controller.js

```diff
@@ -6,6 +6,7 @@ export default class FieldController {
     this.label = config.label;
     this.path = config.path;
     this.type = config.type;
+    this.isPrimaryKey = config.isPrimaryKey;
```

Comment:
We don't want primary key fields to be editable within the Admin UI, so we make sure that setting is easily accessible to allow filtering out later.